MapboxNavigation/NavigationMapView.swift

```diff
@@ -347,20 +355,18 @@ open class NavigationMapView: MLNMapView, UIGestureRecognizerDelegate {
             return
         }
         
-        if !self.tracksUserCourse || self.userAnchorPoint != userCourseView?.center ?? self.userAnchorPoint {
```

Comment:
Why did this boolean check go away?  (TBH I'm not sure I understand what it was doing in the first place)

Reply:
I think this check was in place so centering the puck only happened if it was not centred already (optimisation maybe). As the map is moving during navigation we need to position the puck anyway, thats why I removed it.

Reply:
The map stops moving if the user pans. Does that matter? 

Is this change required for the "position camera in bottom 1/3" of the screen work, or is this some bonus cleanup?

Reply:
But why did it change in this PR? Again, I can't tell if you are doing random cleanup to make things nicer, or if this is required due to other changes you made to get the puck to appear in the bottom 1/3.

Reply:
I did remove this because it makes no difference. This if clause resolved to true in all scenarios I've tested. I guess this was relevant when we manually posited the userCourseView.center to the screen's center. Now that this is gone it's not necessary anymore.